src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -28,7 +28,6 @@
           <div class="kuiLocalSearch">
             <input
               parse-query
-              input-focus
```

Comment:
I don't think removing `input-focus` is actually necessary to accomplish the current experience, I just removed it when I was playing around with things. I left it out because I remember this auto-focus came up as something that hurts accessibility in the accessibility study that was done awhile back. If we want to discuss that outside of this PR, I can try putting it back in.